lib/mayaUsd/ufe/UsdUndoDuplicateCommand.cpp

```diff
@@ -69,19 +72,6 @@ void UsdUndoDuplicateCommand::primInfo(
     // has a numerical suffix, increment it, otherwise append "1" to it.
     auto dstName = uniqueName(childrenNames, srcPrim.GetName());
     usdDstPath = parent.GetPath().AppendChild(TfToken(dstName));
```

Comment:
line 66-73 can be factored out. See uniqueChildName utility function:

https://github.com/Autodesk/maya-usd/blob/91994c05e287ca22d522556eee3d82eb4c93ecc4/lib/mayaUsd/ufe/Utils.cpp#L123

Reply:
Done